### DISCUSSION TITLE
Is it possible to disable MQTT topics related to audio statistics (dBFS and rms)?

### DISCUSSION THREAD

**Discussion by Mincka** [blakeblackshear/frigate#9033] [Ask A Question]
I realised today that my MQTT broker is constantly spammed with messages from the camera with audio enabled.
The number of messages for this useless (for me) information is insane compared to other cameras.
![image](https://github.com/blakeblackshear/frigate/assets/6184289/dda858ca-0277-4270-9990-3c755053751d)
It's about 1 message for each topic (dBFS and rms) per second, per camera. Said like that it does not like much for an MQTT server to handle but it's definitely not needed so I would appreciate being able to disable it, or make it available with some kind of debug flag.

**Comment by NickM-27** [comment]
1 message per second is basically nothing for mqtt as it can handle 100,000+ messages per second. This data is also used in the integration. I don't think in 0.13 there will be an option to disable it, but it is possible in the future that mqtt topics can be more customizable.

**Reply by Mincka** [reply]
I did not say it's a performance issue or something like that. It's just unnecessary noise for me.
For my 5 cameras with audio enabled, that's 10 messages per second that nothing else will ever use.

Where is it used currently in the integration? I've only found stale requests related to that:
https://github.com/blakeblackshear/frigate-hass-integration/issues/549
https://github.com/blakeblackshear/frigate-hass-integration/issues/552

Maybe another idea would be able to configure some kind of threshold to have these audio messages sent only if there's a minimal difference between values. A little bit like it's done for motion detection, we can adjust the sensitivity.

**Reply by NickM-27** [reply]
another user requested (and implemented) this data as it is being used for a graph that shows active audio level. The idea of only updating after a threshold was asked about but the point was brought up that even small changes can be useful when showing active audio data and since this doesn't really present any problems once per second was accepted. 

Currently the integration doesn't have a sensor but there are requests and users are already using it in manual sensors. 

0.13 is almost at the end of the release cycle so I don't see any changes coming in this regard given that this is not an issue, it is only noticeable when actively watching MQTT explorer or something.

**Reply by Mincka** [reply]
Thank you for the explanations. I don't expect anything for 0.13, and even further if this idea does not gain any interest from others.
This was just a question and a few suggestions. I'm completely fine if it's considered as useless or low priority. 🙂 Sorry if I made it look like an issue.

**Reply by letshin** [reply]
Is it perhaps possible to organise the topics under separate  sub-topics eg #/audio/ambient/ to avoid flooding and to enable the use of wildcards?
